### PR TITLE
[Method References] LPS-163772 Convert method references to field references in module portal-search-elasticsearch7-impl | part 8

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I convert method references to field references in module portal-search-elasticsearch7-impl.

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @catarineaguiar  , Could you please help me check the test results ? Thanks!

**Comment by catarineaguiar** [comment]
no regressions.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by W2jx** [comment]
Force-Push  is just to fix the problem mentioned in comment.

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:sf

**Comment by W2jx** [comment]
Hi @zhangyan0701  , Could you please help me check the test results ? Thanks!

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/cluster/ClusterRequestExecutorFixture.java [hunk 1]
```diff
@@ -30,21 +31,22 @@ public void setUp() {
...
+				clusterHealthStatusTranslator, _elasticsearchClientResolver));
+		ReflectionTestUtil.setFieldValue(
+			clusterRequestExecutor, "_stateClusterRequestExecutor",
+			_createStateClusterRequestExecutor(_elasticsearchClientResolver));
+		ReflectionTestUtil.setFieldValue(
+			clusterRequestExecutor, "_statsClusterRequestExecutor",
+			_createStatsClusterRequestExecutor(
+				clusterHealthStatusTranslator, _elasticsearchClientResolver));
+
+		_clusterRequestExecutor = clusterRequestExecutor;
```
Comment (dantewang) [inline modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/cluster/ClusterRequestExecutorFixture.java:49]: @W2jx I'm still seeing such usages in several places in this PR. What I meant is, why not

```
_clusterRequestExecutor = new ElasticsearchClusterRequestExecutor();

ReflectionTestUtil.setFieldValue(_clusterRequestExecutor, xxx, xxx);
```

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/cluster/ClusterRequestExecutorFixture.java (hunk 1 above)
Comment (W2jx) [reply modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/cluster/ClusterRequestExecutorFixture.java:49]: I get it, and I will fix them.

**Comment by W2jx** [comment]
Force-Push is just to fix the problems mentioned in comments.

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
no regressions.

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
@shuyangzhou @tinatian Forwarding this one as all tests passed review and changes are direct replacements.

**Comment by dantewang** [comment]
ci:forward

**Pull request closed without merging**